rhc-ose-ansible/roles/openshift-management/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+openshift_token:
+openshift_master_url: localhost
+openshift_login_insecure_flag: --insecure-skip-tls-verify=true
+openshift_login_insecure: False
+openshift_prune_builds_complete: 5
+openshift_prune_builds_failed: 1
+openshift_prune_builds_keep_younger: 1h0m0s
```

Comment:
Not really an issue but isn't the default of 1hr a bit small of a window? I figured this should be something like a few days. Please explain if its not just a placeholder.